### DISCUSSION TITLE
Store parsed template in redis

### DISCUSSION THREAD

**Discussion by kayix** [harttle/liquidjs#298] [Ideas]
When we parse template it returns javascript tokens. We can save these tokens in js file or in memory. I want to store them in redis database but they are not string. So is there any way to save them as string ? I already tried JSON.stringify but its not working.

**Comment by harttle** [comment]
As commented in https://github.com/harttle/liquidjs/issues/196#issuecomment-594733452, the tokens cannot be serialized currently. And I'm not sure whether it's worthwhile to deserialize instead of parse again. But I'll keep this issue open for more discussion.
